plugins/snopes.py

```diff
@@ -1,34 +1,67 @@
 import re
+from urllib import urlencode
```

Comment:
just import urllib and use urllib.urlencode

Reply:
Done.